src/server/utils/wallets/getLocalWallet.ts

```diff
@@ -16,6 +16,10 @@ interface GetLocalWalletParams {
   walletAddress: string;
 }
 
+/**
+ * @deprecated
+ * DEPRECATED: Use getLocalWalletAccount instead
+ */
```

Comment:
we can't because there's no way to expose the `privateKey` from the `Account` that we get from `getLocalWallet`. lmk if you think it's worth exposing it separately.